app/jobs/tts_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# This jobs generates TTS audio attachments for all words with enabled TTS but without audio attachment.
+# It iterates automatically through all words and ends, when there are no more words to process.
+# There should be a cronjob running this job every 5 minutes and avoid duplicates, to make sure, that the jobs
+# is always running.
+class TtsJob < ApplicationJob
```

Comment:
Do I understand it correctly that this job is supposed to be running like a daemon? That means that we hit the database every second with the query to find all words without an audio attachment, right? It does stop when all all words are "audiofied". In the worst case, when creating a new word, you need to wait for 5min until the audio is created.

Another possibility would be to have an `after_save` callback which invokes a job specifically for that word, which then generates the audio for that word.

This would also allow to inspect successes and failures in the job scheduler itself as every TTS generation would be visible as a single job.

Reply:
Done :)